services/_shared/personas/code_reviewer/claim_check.py

```
@@ -0,0 +1,345 @@
+    settle: int | None = None
+    deep: Bound | None = None
+    for path, source in file_contents.items():
+        if not path.endswith(".py"):
+            continue
+        for m in _SETTLE_CAP_CODE.finditer(source):
+            # Prefer settle-related modules when multiple min(base, N) exist.
+            n = int(m.group(1))
+            if "settle" in path.lower() or "snapshot" in path.lower():
+                settle = n
+            elif settle is None:
+                settle = n
```

Ruff (0.15.21)

[warning] Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)